governance.md

```diff
@@ -0,0 +1,69 @@
+# Brigade Governance
+
+The following document describes how the Brigade project governance operates.
+
+## The Brigade Project
+
+The Brigade project is made up of several codebases with different release cycles. These codebases include:
+
+- Brigade
+- Kashti
+- gateways, libraries, and integrations
+
+There are two levels of maintainers for Brigade: org maintainers and project maintainers.
+
+## Brigade Org Maintainers
+
+The Brigade org maintainers are responsible for:
+
+- Maintaining the mission, vision, values, and scope of the project
+- Refining the governance and charter as needed
+- Making project level decisions
+- Resolving escalated project decisions when responsible project maintainers are blocked
+- Managing the Brigade brand
+- Managing access to Brigade assets such as source repositories, hosting, project calendars
+- Handling code of conduct violations
+- Deciding what sub-groups are part of the Brigade project
+- Overseeing the resolution and disclosure of security issues
+- Managing financial decisions related to the project
+
+The org maintainers have to match the following criteria:
+
+- There will be between 3 and 5 people with a diverse representation from multiple companies
+- Any project maintainer is eligible for a position as an org maintainer
+- No one company or organization can employ a simple majority of the org maintainers
+- Org maintainers MUST remain active on the project. If they are unresponsive for > 3 months, they will lose org maintainership, unless a [super-majority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) of the other org maintainers agrees to extend the period to be greater than 3 months
+
+Changes to the org maintainers use the following:
+- An org maintainer may step down by emailing the org maintainers mailing list
+- When there is an opening for a new org maintainer, any contributor to any Brigade project may nominate a suitable maintainer as replacement
+    - The nomination period will be one month starting the day after the position becomes available
+    - The nomination must be made via the Brigade mailing list
```

Comment:
Do we have one of these?

Reply:
Not yet -- added an item to the list of things we need for the new org -- https://github.com/Azure/brigade/issues/789